sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLPoolDAO.java

```diff
@@ -849,7 +849,7 @@ public Pool mapRow(ResultSet rs, int rowNum) throws SQLException {
         }
 
         if (!isLazy()) {
-          p.setExperiments(experimentDAO.listByPoolId(id));
+          // BATS Hibernate will load this
```

Comment:
BATS is a sad story -- hibernating kills bats (if they're infected with white-nose fungus), just like Hibernatizing kills this code.